### ISSUE TITLE
Need help in transcription 

### ISSUE THREAD

**Issue by BibekBazaz** [marytts/marytts#504]
Hello Everyone,

I am developing a Hindi TTS using marytts.

I have build a allophone set for Hindi  and a list of words for training. I have accordingly written the allophones for each word in the wordfile corresponding to their hindi spellings and feed it to transcription gui.

but only a handful get trained and others are rejected. also a java exception is shown.
the complete log file is :
prev: Serif
next: Serif
alignment ...
iteration 0
iteration 1
iteration 2
iteration 3
iteration 4
alignment completed.
training ...
      Training decision tree for: ं
      Training decision tree for: े
      Training decision tree for: त
      Training decision tree for: थ
      Training decision tree for: द
      Training decision tree for: ु
      Training decision tree for: न
      Training decision tree for: ए
      Training decision tree for: म
      Training decision tree for: ै
      Training decision tree for: य
      Training decision tree for: ल
      Training decision tree for: ऐ
      Training decision tree for: व
      Training decision tree for: ख
      Training decision tree for: क
      Training decision tree for: श
      Training decision tree for: ह
      Training decision tree for: स
      Training decision tree for: ि
      Training decision tree for: null
      Training decision tree for: ज
      Training decision tree for: ा
training completed.
java.lang.IllegalArgumentException: Cannot syllabify empty phone string
    at marytts.modules.phonemiser.AllophoneSet.syllabify(AllophoneSet.java:479)
    at marytts.modules.phonemiser.TrainedLTS.syllabify(TrainedLTS.java:176)
    at marytts.tools.transcription.TranscriptionTable.trainPredict(TranscriptionTable.java:221)
    at marytts.tools.transcription.TranscriptionGUI.trainPredictActionPerformed(TranscriptionGUI.java:816)
    at marytts.tools.transcription.TranscriptionGUI.access$8(TranscriptionGUI.java:810)
    at marytts.tools.transcription.TranscriptionGUI$8.actionPerformed(TranscriptionGUI.java:372)
    at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2018)
    at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2341)
    at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
    at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
    at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
    at java.awt.Component.processMouseEvent(Component.java:6516)
    at javax.swing.JComponent.processMouseEvent(JComponent.java:3312)
    at java.awt.Component.processEvent(Component.java:6281)
    at java.awt.Container.processEvent(Container.java:2229)
    at java.awt.Component.dispatchEventImpl(Component.java:4872)
    at java.awt.Container.dispatchEventImpl(Container.java:2287)
    at java.awt.Component.dispatchEvent(Component.java:4698)
    at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4832)
    at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4492)
    at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4422)
    at java.awt.Container.dispatchEventImpl(Container.java:2273)
    at java.awt.Window.dispatchEventImpl(Window.java:2719)
    at java.awt.Component.dispatchEvent(Component.java:4698)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:747)
    at java.awt.EventQueue.access$300(EventQueue.java:103)
    at java.awt.EventQueue$3.run(EventQueue.java:706)
    at java.awt.EventQueue$3.run(EventQueue.java:704)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:77)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
    at java.awt.EventQueue$4.run(EventQueue.java:720)
    at java.awt.EventQueue$4.run(EventQueue.java:718)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:77)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:717)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
Testing FST...
Testing complete. 19 entries (19 correct, 0 failed
[allophones.hi_IN.txt](https://github.com/marytts/marytts/files/179144/allophones.hi_IN.txt)

**Comment by seblemaguer** [comment]
duplicate of #502. Please do not open a new issue for exactly the same problem than before. People will answer when they will be able to do it.
